client/src/App.jsx

```diff
@@ -16,6 +19,7 @@ function App() {
           <Route path="/" element={<Login />} />
           <Route path="/forum" element={<Forum />} />
           <Route path="/dashboard" element={<Dashboard />} />
+          <Route path="/footer" element={<FooterPage />} />
```

Comment:
In principle the footer component would not have a specific route path, right? I understand this is just temporary to easily check how the footer looks while building it. And that when ready, the footer would not be here, but rendered as part of other pages/components. Just double-checking :)

Reply:
Yep - just to view it for now whilst playing around, then once we're assembling everything it will sit at the bottom of every page.